### DISCUSSION TITLE
[Feature] Ask for confirmation when deleting from an album

### DISCUSSION THREAD

**Discussion by erikvanoosten** [immich-app/immich#9855] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

When you select a number of photos in an album, and then press the move-to-bin button, the photos will be gone immediately. To prevent accidental loss, a confirmation should be shown. Ideally, the confirmation dialog should say something like 'Delete 10 photos from album X?'.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by blublub** [comment]
yeah a better seperation from "remove from album" and delete/move image to trash would be useful

**Comment by Jeff5496** [comment]
I second this request for Mobile and Web!

**Comment by Deiiv** [comment]
This is also an issue when viewing single photos in the "Photos" section with an external library (even though it can't delete those files anyways since it's set to read-only). It would be great to have a confirmation to delete since I already accidentally did it twice (the delete is right next to the "back" button on Android).

An additional change could be to add an option to fully hide the delete menu option for those of us who have read-only libraries.
